### ISSUE TITLE
Add NAD as shortName for NetworkAttachmentDefinition

### ISSUE THREAD

**Issue by yossisegev** [k8snetworkplumbingwg/multi-net-spec#7]
Currently NetworkAttachmentDefinition has only "net-attach-def" as a short-name. This doesn't really make life easier, as we use this CRD kind quite a lotin CLI ("oc get network-attachment-definition", etc.).
Can you please add "nad" as a short-name?
Thanks.

**Comment by ormergi** [comment]
I agree, `nad` is way more convenient then `net-attach-def`, hope it will be accepted :pray:
